test/e2e/facility-helpers.js

```diff
@@ -0,0 +1,267 @@
+const mock = require('./mock-helpers');
+
+/* eslint-disable camelcase */
```

Comment:
Yes, good catch. We should do that for consistency. I can roll that into the PR for the appointment stats

https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2859